src/client/eppo-precomputed-client.ts

```diff
@@ -51,32 +67,38 @@ export default class EppoPrecomputedClient {
   private precomputedFlagsRequestParameters?: PrecomputedFlagsRequestParameters;
   private subjectKey?: string;
   private subjectAttributes?: Attributes;
+  private flagKeySalt = '';
+  private precomputedFlagStore: IConfigurationStore<PrecomputedFlag>;
+  private isObfuscated: boolean;
 
-  constructor(
-    private precomputedFlagStore: IConfigurationStore<PrecomputedFlag>,
-    private isObfuscated = false,
-  ) {}
+  constructor(options: EppoPrecomputedClientOptions) {
+    this.precomputedFlagStore = options.precomputedFlagStore;
+    this.isObfuscated = options.isObfuscated ?? true; // Default to true if not provided
+  }
 
-  public setPrecomputedFlagsRequestParameters(
-    precomputedFlagsRequestParameters: PrecomputedFlagsRequestParameters,
-  ) {
-    this.precomputedFlagsRequestParameters = precomputedFlagsRequestParameters;
+  public setIsObfuscated(isObfuscated: boolean) {
```

Comment:
This shouldn't be public and generally should not exist at all. Obfuscation should only be determined by the received configuration